File: poo/1. Calculating an Employee's Salary JS/discount.java
public class discount extends salary {
    private double health;
    private double pension;
    private double arl;
    private double discount;

    public discount(double health, double pension, double arl, int salario) {
        super(1080292891, "Luis", "Duarte", salario);
        this.health = health;
        this.pension = pension;
        this.arl = arl;
    }
    public double getHealth() {
        return health;
    }
    public double getPension() {
        return pension;
    }
    public double getArl() {
        return arl;
    }
 
    @Override
    public double payTotal(){
        this.discount = (this.getSalario() * health) + (this.getSalario() * pension) + (this.getSalario() * arl);
        return this.discount;
    }
}

File: poo/1. Calculating an Employee's Salary/discount.php
<?php
class discount extends salary {
    private float $health;
    private float $pension;
    private float $arl;
    private float $discount;

    public function __construct(float $health, float $pension, float $arl, int $salario) {
        parent::__construct(1080292891, "Luis", "Duarte", $salario);
        $this->health = $health;
        $this->pension = $pension;
        $this->arl = $arl;
    }
    public function getHealth(): float {
        return $this->health;
    }
    public function getPension(): float {
        return $this->pension;
    }
    public function getArl(): float {
        return $this->arl;
    }
    public function payTotal(): float {
        $this->discount = ($this->getSalario() * $this->health) + ($this->getSalario() * $this->pension) + ($this->getSalario() * $this->arl);
        return $this->discount;
    }
}
?>